app/mixins/ui-kit/border.js

```diff
@@ -0,0 +1,51 @@
+import Mixin from '@ember/object/mixin';
+import { or } from '@ember/object/computed';
+import { checkDictionary } from 'travis/utils/ui-kit/assertions';
+import prefix from 'travis/utils/ui-kit/prefix';
+import concat from 'travis/utils/ui-kit/concat';
+
+// Border
+const BORDER_WIDTHS = {
+  NONE: 'none',
+  ONE: 1,
```

Comment:
I'd suggest to either stick with general `xs|sm|md|lg` pattern for border widths or use something more abstract, like `none|thin|thick`.

Reply:
Makes sense to me, change made :+1: